HookTest/units.py

```diff
@@ -390,32 +455,39 @@ def unique_passage(self):
         except Exception:
             yield False
 
-
     def has_urn(self):
         """ Test that a file has its urn saved
         """
         if self.xml is not None:
             if self.scheme == "tei":
-                urns = self.xml.xpath("//tei:text[starts-with(@n, 'urn:cts:')]", namespaces=TESTUnit.NS)
```

Comment:
Ah, I see #35. Ok, not sure really which is right here, but if we have had that in the guidelines, I agree the test should at least match.